website/docs/concepts/provider_lifecycles.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Provider Lifecycles
+---
+
+## When does my Provider get created and disposed?
+
+The states that all different types of providers can go through are the same:
+
+- Uninitialized
+- Creating
+- Alive
+- Disposing
+
+### Uninitialized
+
+An uninitialized provider does not take up any memory since its state is not initialized.
+It will stay that way until an Alive provider or the UI reads or watches it.
```

Comment:
> It will stay that way until [...]


What is that supposed to mean? Providers never go back to initialized state, besides maybe autoDispose